content/chainguard/chainguard-images/working-with-images/scanners/grype-tutorial/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Using the grype Container Scanner"
+type: "article"
+linktitle: "grype-tutorial"
+aliases: 
+- /chainguard/chainguard-images/working-with-images/scanners/grype-tutorial
+description: "Learn to use grype to detect CVEs in images"
+date: 2024-06-06:08:00+02:00
+lastmod: 2024-06-06:08:00+00:00
+tags: ["Conceptual", "CVE"]
+draft: false
+images: []
+menu:
+  docs:
+    parent: "scanners"
+weight: 100
+toc: true
+---
+
+[grype](https://github.com/anchore/grype) is a vulnerability scanner for container images and filesystems developed and maintained by [Anchore](https://anchore.com/) and written in the Go programming language. grype can scan from Docker, OCI, Singularity, podman, image archives, and local directory. grype is compatible with SBOMs generated by [syft](https://github.com/anchore/syft), and grype's [vulnerability database](https://github.com/anchore/grype-db) draws from a wide variety of sources, including [Wolfi SecDB]( https://github.com/wolfi-dev/wolfictl/blob/main/docs/cmd/wolfictl_advisory_secdb.md).
```

Comment:
Hm, my read on the project is that they are going for lower case, their title in the project is lower case, and I do broadly feel like we should try to respect their choice. I agree that they're not always the best about it themselves, but reading between the lines I do think they try to be lower case, you see it lower case where it would be capitalized in their blogs and such. I do find this kind of painful, since it looks a bit ugly at the start of sentences, but I think it's most correct, and our style says to spell projects the weird way they want to be spelled (I think). Open to more pushback here, though, let me know what you think.

Reply:
Well, I dug a little deeper and, to your point, I found at least three blog posts where they consistently capitalize the project names:
* [ex. 1](https://anchore.com/blog/say-goodbye-to-false-positives/)
* [ex. 2](https://anchore.com/blog/finding-and-fixing-the-jsonwebtoken-vulnerabilities/)
* [ex. 3](https://anchore.com/blog/four-years-of-syft-development-in-4-minutes-at-4k/) (this example is mostly Syft but the same point applies)

so :shrug: it seems to me like the GH project is what's out of line with their style guide and I'm inclined to push back more. But we can also throw the question to the team if you aren't convinced

Reply:
No, this is evidence enough. Kind of annoying that they're that inconsistent. I'll swap them out, thanks for taking the time to dig up the posts! Honestly it will make it look better, the lower case looks kind of stupid.

Reply:
OK, now I'm semantically satiated on the word Grype, but the deed is done. Ditto for Syft.